patches/minecraft/net/minecraft/enchantment/EnchantmentHelper.java.patch

```diff
@@ -1,6 +1,25 @@
 --- ../src-base/minecraft/net/minecraft/enchantment/EnchantmentHelper.java
 +++ ../src-work/minecraft/net/minecraft/enchantment/EnchantmentHelper.java
-@@ -302,7 +302,7 @@
+@@ -143,12 +143,14 @@
+         return field_77520_b.field_77497_a;
+     }
+ 
++    /**
++     * Does nothing
++     * Use {@link EnchantmentHelper#getModifierForCreature(ItemStack, Entity)} instead
++     */
++    @Deprecated
+     public static float func_152377_a(ItemStack p_152377_0_, EnumCreatureAttribute p_152377_1_)
+     {
+-        field_77521_c.field_77495_a = 0.0F;
+-        field_77521_c.field_77494_b = p_152377_1_;
+-        func_77518_a(field_77521_c, p_152377_0_);
+-        return field_77521_c.field_77495_a;
++        return 0.0F;
```

Comment:
I'm not sure I see the purpose of changing this.
Deprecating it is fine, but why return 0?